terragrunt/aws/config/wafv2_logs.tf

```diff
@@ -0,0 +1,243 @@
+#
+# Config Rule
+#
+resource "aws_config_config_rule" "cbs_wafv2_logs_rule" {
+  name        = "cbs_wafv2_logs_rule"
+  description = "Checks that the WAFV2 ACL's are logging to either S3 or Kinesis."
+
+  source {
+    owner             = "CUSTOM_LAMBDA"
+    source_identifier = aws_lambda_function.cbs_wafv2_logs_rule.arn
+
+    source_detail {
+      message_type                = "ScheduledNotification"
+      maximum_execution_frequency = var.config_max_execution_frequency
+    }
+  }
+
+  scope {
+    compliance_resource_types = ["AWS::WAFv2::WebACL"]
+  }
+
+  depends_on = [
+    aws_lambda_permission.cbs_wafv2_logs_rule,
+  ]
+}
+
+#
+# Lambda function used by the ConfigRule
+#
+data "archive_file" "cbs_wafv2_logs_rule" {
+  type        = "zip"
+  source_file = "config_rules/wafv2_logs/webacl_logs_rule.py"
+  output_path = "/tmp/cbs_wafv2_logs_rule.zip"
+}
+
+resource "aws_lambda_function" "cbs_wafv2_logs_rule" {
+  filename      = "/tmp/cbs_wafv2_logs_rule.zip"
+  function_name = "cbs_wafv2_logs_rule"
+  role          = aws_iam_role.cbs_wafv2_logs_rule.arn
+  handler       = "webacl_logs_rule.lambda_handler"
+
+  source_code_hash = data.archive_file.cbs_wafv2_logs_rule.output_base64sha256
+  runtime          = "python3.9"
+
+  environment {
+    variables = {
+      FIREHOSE_ARN = aws_kinesis_firehose_delivery_stream.cbs_default_stream.arn
+    }
+  }
+
+  tracing_config {
+    mode = "PassThrough"
+  }
+
+  tags = {
+    (var.billing_tag_key) = var.billing_tag_value
+    Terraform             = true
+  }
+}
+
+resource "aws_lambda_permission" "cbs_wafv2_logs_rule" {
+  action        = "lambda:InvokeFunction"
+  function_name = aws_lambda_function.cbs_wafv2_logs_rule.arn
+  principal     = "config.amazonaws.com"
+  statement_id  = "AllowExecutionFromConfig"
+}
+
+resource "aws_cloudwatch_log_group" "cbs_wafv2_logs_rule" {
+  name              = "/aws/lambda/${aws_lambda_function.cbs_wafv2_logs_rule.function_name}"
+  retention_in_days = 14
+
+  tags = {
+    (var.billing_tag_key) = var.billing_tag_value
+    Terraform             = true
+  }
+}
+
+#
+# Lambda execution role
+#
+resource "aws_iam_role" "cbs_wafv2_logs_rule" {
+  name               = "cbs_wafv2_logs_rule"
```

Comment:
Same comment about name prefix with a `Cbs` or `cbs-`.

Also totally fine adding a `cbs_` prefix exemption though to our AWS nuke config.